python/Ganga/Runtime/bootstrap.py

```diff
@@ -1279,7 +1279,8 @@ def launch_IPython(local_ns, args, error_handler, ganga_prompt):
         ipshell.confirm_exit = config['confirm_exit']
 
         # Launch embedded shell
-        ipshell(local_ns=local_ns)
+        from Ganga import GPI
+        ipshell(local_ns=local_ns, module=Ganga.GPI)
```

Comment:
I think that probably `local_ns` should be `Ganga.GPI`+`~/.ganga.py`+`config['StartupGPI']` but this needs some thought.